tests/DataFixtures/ORM/resources/products.yml

```diff
@@ -1,36 +1,54 @@
+Sylius\Component\Core\Model\ProductTranslation:
+    productTranslation1:
+        slug: mug-star-wars
+        locale: en
+        name: 'Star wars mug'
+        description: <paragraph(2)>
+        translatable: "@product1"
+    productTranslation2:
+        slug: star-wars-mug
+        locale: en_US
+        name: 'Star Wars mug'
+        description: <paragraph(2)>
+        translatable: "@product1"
+
+
+Sylius\Component\Core\Model\ProductVariant:
+    productVariant:
+        code: MUG_1
+        product: "@product2"
+
 Sylius\Component\Core\Model\Product:
     product1:
         updatedAt: 2015-10-10
-        currentLocale: en_US
-        currentTranslation: "@productTranslation1"
+        fallbackLocale: en_US
+        currentLocale: en
         code: MUG_SW
+        translations:
+            - "@productTranslation1"
+            - "@productTranslation2"
     product2:
         updatedAt: 2015-10-04
         currentLocale: en_US
-        currentTranslation: "@productTranslation2"
+        fallbackLocale: en_US
         code: MUG_LOTR
+        slug: mug-lotr
+        name: 'Lotr mug'
+        description: <paragraph(2)>
+        variants:
+            - "@productVariant"
     product3:
         updatedAt: 2015-10-05
         currentLocale: en_US
-        currentTranslation: "@productTranslation3"
+        fallbackLocale: en_US
         code: MUG_BB
-
-Sylius\Component\Core\Model\ProductTranslation:
-    productTranslation1:
-        slug: mug-star-wars
-        locale: en_US
-        name: 'Star wars mug'
-        description: <paragraph(2)>
-        translatable: "@product1"
-    productTranslation2:
-        slug: mug-lotr
-        locale: en_US
-        name: 'Lotr mug'
-        description: <paragraph(2)>
-        translatable: "@product2"
-    productTranslation3:
         slug: mug-breaking-bad
-        locale: en_US
         name: 'Breaking bad mug'
         description: <paragraph(2)>
-        translatable: "@product3"
+    product{4..22}:
```

Comment:
Maybe we should put them to other file? It will reduce number of require database inserts. 
e.g. `producs.yml` and `many_products.yml`